packages/kernel-5.10/config-bottlerocket-aws

```diff
@@ -0,0 +1,6 @@
+# Support boot from IDE disks
+CONFIG_ATA=y
+CONFIG_ATA_GENERIC=y
```

Comment:
Do we want the generic driver? We do not ship it with the metal variants.
If we actually want it, we probably also want it on metal.

Reply:
On a second look, indeed, this won't be needed. I'll drop it in the next revision.